.github/workflows/native_s3_fasttext.yml

```diff
@@ -10,14 +10,14 @@ permissions:
 jobs:
   build-fasttext-jni-linux:
     runs-on: ubuntu-latest
-    container: amazonlinux:2
+    container: amazonlinux:2023
```

Comment:
We cannot upgrade to amazonlinux:2023 yet

Reply:
All internal AWS customers are using AL2

Reply:
OK. Changed the checkout action.